packages/context/src/objects/objects.ts

```diff
@@ -43,6 +43,37 @@ export type ObjectInstance = {
     [key: string]: unknown;
 };
 
+export type FormioInputTypes = {
+    label: string;
+    type: string;
+    key: string;
+    initialValue?: any;
+    property?: Property;
+    input?: boolean;
+    placeholder?: string;
+    description?: string;
+    tooltip?: string;
+    prefix?: string;
+    suffix?: string;
+    showCharCount?: boolean;
+    readOnly?: boolean;
+    isMultiLineText?: boolean;
+    widget?: string;
+    tableView?: boolean;
+    verticalLayout?: boolean;
+    data?: {
+        resource?: string;
+        values?: [];
+    };
+    dataSrc?: string;
+    validate?: {
+        required: boolean;
+        operator: any;
```

Comment:
Does this correspond to the "Must match regular expression rules" validation setting in Builder?  If so, I assume we know the type, it shouldn't be `any`.  Also, I don't see where you specify the validation regular expressions that go with it.

Reply:
Added the regexes [].